tools/local-network/local_network.py

```diff
@@ -580,6 +589,14 @@ def default_entry_point(self, network_type, skip_build=False, block_version=None
                 peers = [Peer(p) for p in other_nodes]
                 self.add_node(str(i), peers, QuorumSet(3, other_nodes))
 
+        elif network_type == 'dense3':
+            #  3 node interconnected network requiring 2 out of 3 nodes.
+            num_nodes = 3
+            for i in range(num_nodes):
+                other_nodes = [str(j) for j in range(num_nodes) if i != j]
+                peers = [Peer(p) for p in other_nodes]
+                self.add_node(str(i), peers, QuorumSet(1, other_nodes))
```

Comment:
the quorum sets are "self + 1", so this effectively means that 2 nodes have to validate anything that happens. So that's what we want in a 2 out of 3 configuration